tests/test_modules/test_dataset_analysis/test_dataset_warnings.py

```diff
@@ -115,16 +116,27 @@ def remove_one_cls(mod):
     mod.get_dataset_split_manager(DatasetSplitName.eval)._base_dataset_split = augmented
 
 
-def test_with_dataset_and_indices(simple_text_config):
-    with pytest.raises(ValueError):
-        DatasetWarningsModule(
-            dataset_split_name=DatasetSplitName.eval,
-            config=simple_text_config,
-        )
+def test_dataset_warnings_with_one_ds(tiny_text_config_one_ds):
+    split, _ = get_tiny_text_config_one_ds_name(tiny_text_config_one_ds)
+    generate_mocked_dm(tiny_text_config_one_ds, dataset_split_name=split)  # Needed for word count.
 
-    with pytest.raises(ValueError):
-        DatasetWarningsModule(
-            dataset_split_name=DatasetSplitName.all,
-            config=simple_text_config,
-            mod_options=ModuleOptions(indices=[1, 2]),
-        )
+    mod = DatasetWarningsModule(
+        dataset_split_name=DatasetSplitName.all,
+        config=tiny_text_config_one_ds,
+    )
+    output = mod.compute_on_dataset_split()[0].warning_groups
+
+    general_warnings = output[0]
+    assert len(general_warnings.warnings) == 2, "Only 2 general warnings with one dataset split"
+    assert all(
+        [
+            len(comparison.data) == 1
+            for warning in general_warnings.warnings
+            for comparison in warning.comparisons
+        ]
+    ), "Only data for one split"
+
+    syntactic_warnings = output[1]
+    assert not all(
+        [comparison.alert for comparison in syntactic_warnings.warnings[0].comparisons]
+    ), "No alert with one split"
```

Comment:
Did you mean `not any()`? So that any one alert would make the test fail?
```suggestion
    assert not any(
        comparison.alert for comparison in syntactic_warnings.warnings[0].comparisons
    ), "No alert with one split"
```
Another way to write according to De Morgan's law:
```suggestion
    assert all(
        not comparison.alert for comparison in syntactic_warnings.warnings[0].comparisons
    ), "No alert with one split"
```